lib/fastlane_core/command_executor.rb

```diff
@@ -59,6 +59,9 @@ def execute(command: nil, print_all: false, print_command: true, error: nil, pre
             end
             Process.wait(pid)
           end
+        rescue Errno::EIO
+          # This is expected on some linux systems, that indicates that the subcommand finished
+          # and we kept trying to read, ignore it
```

Comment:
The examples you referenced have been catching the `Errno::EIO` inside of the `PTY.spawn` block. Do you think it's important to catch it in that context?